libbeat/docs/shared-path-config.asciidoc

```diff
@@ -115,11 +115,11 @@ This can either be set in the config, or with the `--system.hostfs` CLI flag. Th
 ifeval::["{beatname_lc}"=="metricbeat"]
 This is also used by the system module to read files from `/proc` and `/sys`.
 endif::[]
-
+This option is deprecated and will be removed in a future release. To set the filesystem root, use the `hostfs` flag inside the module-level config.
```

Comment:
It seems that this is already removed from other beats in this PR, should we put the whole docs of this setting inside a `ifeval::["{beatname_lc}"=="metricbeat"]`?

Reply:
It's really only used in metricbeat as far as I'm aware, so I'm not sure what should really go where here?

Reply:
This doc file is shared along all beats, I mean to conditionally include the docs for this flag only for Metricbeat if it is going to be already removed in the rest of beats, something like this:

```
ifeval::["{beatname_lc}"=="metricbeat"]
[float]
==== `system.hostfs`
Specifies the mount point of the host's filesystem for use in monitoring a host.
This can either be set in the config, or with the `--system.hostfs` CLI flag. This is used for cgroup self-monitoring.

This is also used by the system module to read files from `/proc` and `/sys`.

This option is deprecated and will be removed in a future release. To set the filesystem root, use the `hostfs` flag inside the module-level config.

Example:

[source,yaml]
------------------------------------------------------------------------------
system.hostfs: /mount/rootfs
------------------------------------------------------------------------------
```
endif::[]